src/control/Control.Layers.js

```diff
@@ -165,7 +165,10 @@ export const Layers = Control.extend({
 
 	// @method collapse(): this
 	// Collapse the control container if expanded.
-	collapse() {
+	collapse(e) {

```

Comment:
Nitpicking: Please use `ev` instead of `e`.

Whenever I see `e` as a parameter, I don't know if that's an `ev`ent, an `err`or or an `ex`ception.